### ISSUE TITLE
About the choice of batch size and learning rate

### ISSUE THREAD

**Issue by syc11-25** [chuanyangjin/fast-DiT#13]
When I train on 8 * A100-80G GPUs, my batchsize can be set to 51200. Does the learning rate need to be adjusted accordingly? Will setting such a large batchsize affect training?

**Comment by syc11-25** [comment]
The model I trained is S/2
